examples/cpu-partitioning.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: openshift-cpu-partitioning
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Configuration for cpu-partitioning
+      include=openshift-node,cpu-partitioning
+      [variables]
+      isolated_cores=1-3
```

Comment:
Created problems on my dual core worker node, perhaps we should use a smaller range for this example?

Reply:
Thanks for the report!